joomla/joomla-cms
# [4.0] Transition of Joomla's Wrapper module and component from HTML4 to HTML5

Pull Request for Issue # .

### Summary of Changes
Made a transition in Joomla's WRAPPER Module and Component from HTML4 to HTML5 standard.
### Testing Instructions
Using "px" or "%" in "height" field (required by HTML5 standard).

If scrollbars do appear after selecting "No" option in "Scroll Bars" while creating wrapper element, you need to change the dimensions of wrapper element (ie. height and/or width of wrapper element).
### Expected result

To work flawlessly.

### Documentation Changes Required
Need to modify 6 files to work without any problems or hiccups:

- modules/mod_wrapper/mod_wrapper.xml

- modules/mod_wrapper/mod_wrapper.php

- modules/mod_wrapper/Helper/WrapperHelper.php

- modules/mod_wrapper/tmpl/default.php

- components/com_wrapper/tmpl/wrapper/default.xml

- components/com_wrapper/tmpl/wrapper/default.php
